### DISCUSSION TITLE
Help! My repository language is wrong!

### DISCUSSION THREAD

**Discussion by lildude** [github-linguist/linguist#5159] [Classification]
Thank you for considering asking the community to help you explain and understand the language analysis of your repository, but before you do, it would be incredibly helpful for everyone if your reviewed the following useful resources first:

- [How Linguist Works](https://github.com/github/linguist/blob/master/docs/how-linguist-works.md),
- the [Troubleshooting](https://github.com/github/linguist/blob/master/docs/troubleshooting.md) steps,
- and consider implementing an [override](https://github.com/github/linguist/blob/master/docs/overrides.md).
 
Reviewing and understanding how Linguist works and how you can control some of its behaviour helps ensure everyone is on the same page when it comes to offering assistance.

[Installing](https://github.com/github/linguist#installation) and [analysing your repository locally](https://github.com/github/linguist#command-line-usage) is the best way to see precisely why Linguist is reporting the language statistics you see on GitHub.com.

If you still need help, please feel free to start a new discussion in the "Language Analysis" category using the template below:

```markdown
## Problem description
<!-- Tell us about the behaviour you are seeing. -->

### URL of the affected repository:
<!-- Being able to see what you're seeing is the first step to helping you -->

### Last modified on:
<!-- YYYY-MM-DD --> 

### Expected language:
<!-- expected language -->

### Detected language:
<!-- Detected language -->

```

**Comment by Sainan** [comment]
Some links here are outdated, because you seem to have moved them from the readme to seperate files.

**Reply by lildude** [reply]
Updated. Thanks for the reminder.
